api/api-doc/storage_service.json

```diff
@@ -1047,7 +1047,7 @@
          ]
       },
       {
-         "path":"/storage_service/cleanup_all",
+         "path":"/storage_service/cleanup_all/{global}",
```

Comment:
Adding this path parameter seems to be a breaking change. I'm not sure if we can backport it. Perhaps we should preserve the default behavior. We can for example add an optional query parameter. On the other hand, probably no one uses this endpoint outside nodetool.

Reply:
Exactly. Only the nodetool we ship use this endpoint and it this compiled into the Scylla itself, so you cannot mismatch versions.

Reply:
We shouldn't introduce breaking changes into public APIs, unless there is a very good reason for it (workaround is very painful). If there is a way to avoid it, we should.